Tools/Sources/needletools/Utilities/Version.swift

```diff
@@ -94,10 +97,10 @@ class Version: Comparable {
         if lhs.major > rhs.major {
             return false
         }
-        if lhs.minor > rhs.minor {
+        if lhs.major == rhs.major && lhs.minor > rhs.minor {
```

Comment:
OK, given this bug.  I think it might be better for us to use the `Version` struct from the `Utility` module. It has this compare code in it and more.

Reply:
Sounds good. I'll switch this over.